pystac/validation/stac_validator.py

```diff
@@ -246,22 +269,18 @@ def validate_extension(
         Returns:
            str: URI for the JSON schema that was validated against, or None if
                no validation occurred.
+
+        Raises:
+           STACValidationError if stac_dict is not valid.
```

Comment:
```suggestion
           STACValidationError if stac_dict is not valid. The exception is raised from the "best" error, as determined by the jsonschema library. To access all jsonschema validation errors, use ``STACValidationError.source``.
```

Formatting isn't quite right here, and the language could use some work, but I think we need to tell people about this new capability.

Reply:
I'm no Tom Waits, but that wording sounds pretty good to me.